doc/design/web.md

```diff
@@ -0,0 +1,141 @@
+# Web Interface design
+
+This design doc will talk about features and web pages needed to let users to manage cloud paddle jobs.
+
+## Feature List
+
+- Account Management
+    - Registration, send email to inform if registration suceeded
+    - Account Login/Logout
+    - Password changing, find back
+    - Download SSL keys
+- Jupiter Notebook
+    - Private Jupiter Notebook environment to run Python scripts
+    - Private workspace
+    - Submit job from Jupiter Notebook
+- Job Dashboard
+    - Job history and current running jobs
+    - Performance Monitoring
+    - Quota Monitoring
+- Datasets
+    - Public Dataset viewing
+    - Upload/Download private datasets
+    - Share datasets
+- Models
+    - Upload/Download models file
+    - Share/Publish Models
+- Paddle Board
+    - Training metrics visualization
+        - cost
+        - evaluator
+        - user-defined metrics
+- Serving
+    - Submit serving instances
+    - Deactivate serving
+    - Serving performance monitoring
+
+## Account Management
+
+I'll skip this section because it is a design that almost every website need.
```

Comment:
If this document is for Production Team's reference, we should at least give an example Web site here. 

In my mind, we do need a mockup for this page.  At least, once a user logs in, s/he must be able to see all his/her jobs listed.  And s/he should be able to click each job to see the job's dashboard.